packages/react-storage/src/components/StorageBrowser/views/LocationActionView/UploadControls.tsx

```diff
@@ -196,6 +182,8 @@ export const UploadControls = (): JSX.Element => {
     { prefix: path, preventOverwrite }
   );
 
+  // Replace with stuff
```

Comment:
Depends on the stuff 😅

Reply:
Oops I have no idea what stuff this was even meant to be now. I'll just remove the comment